### PR TITLE
add --namespace option to Helm template command

### LINKED ISSUE #190: `helm template` command is not setting the `--namespace` option
The `helm template` command does not set the `--namespace` option.
https://github.com/bakdata/kpops/blob/b340ad5b926580f5b0b07d4a7fd82f79ffbc0ddc/kpops/component_handlers/helm_wrapper/helm.py#L156-L163
This leads to a wrong output of the template whenever the helm chart contains `{{ .Release.Namespace }}`

### PR DESCRIPTION
**Pull request by raminqaf**
Fixes #190

### CODE REVIEW

File: kpops/component_handlers/helm_wrapper/helm.py [hunk 1]
```diff
@@ -143,6 +144,7 @@ def template(
 
         :param str release_name: the release name for which the command is ran
         :param str chart: Helm chart to be templated
+        :param str namespace: The namespace the command should execute in
```
Comment (disrupted) [inline kpops/component_handlers/helm_wrapper/helm.py:147]:
```suggestion
        :param str namespace: The Kubernetes namespace the command should execute in
```

Comment (sujuka99) [inline kpops/component_handlers/helm_wrapper/helm.py:147]:
```suggestion
        :param namespace: The namespace the command should execute in
        :type namespace: str
```

File: kpops/component_handlers/helm_wrapper/helm.py [hunk 2]
```diff
@@ -143,6 +144,7 @@ def template(
 
         :param str release_name: the release name for which the command is ran
         :param str chart: Helm chart to be templated
```
Comment (sujuka99) [inline kpops/component_handlers/helm_wrapper/helm.py:146]:
```suggestion
        :param release_name: the release name for which the command is ran
        :type release_name: str
        :param chart: Helm chart to be templated
        :type chart: str
```

File: kpops/component_handlers/helm_wrapper/helm.py [hunk 3]
```diff
@@ -143,6 +144,7 @@ def template(
...
+        :param str namespace: The namespace the command should execute in
         :param dict[str, str] values: `values.yaml` to be used
         :param flags: the flags to be set for `helm template`
         :type flags: HelmTemplateFlags
```
Comment (sujuka99) [inline kpops/component_handlers/helm_wrapper/helm.py:150]:
```suggestion
        :param flags: the flags to be set for `helm template`, defaults to HelmTemplateFlags()
        :type flags: HelmTemplateFlags, optional
```

**Review by sujuka99 — CHANGES_REQUESTED** [review]
Looks good! Suggested improvements to the docstring.

**Review by sujuka99 — APPROVED** [review]

**Pull request merged**